dev/custom-pages/academics-search.scss

```diff
@@ -316,4 +315,76 @@
       }
     }
   }
+}
+
+#contentcol .resources-section {
+  padding-bottom: 35px;
+}
+
+#contentcol .callout {
+  background: #fc0;
+  margin: 0 auto;
+  text-align: center;
+  padding: 48px 0;
+
+  &:last-child {
+    margin-right: 0;
+  }
+
+  .container {
+    vertical-align: top;
+  }
+
+  .icon-link {
+    display: inline-block;
+    margin: 10px auto;
+    width: 25%;
+    vertical-align: top;
+
+    @media (max-width: $screen-xs-max) {
+      width: 50%;
+    }
+
+    a {
```

Comment:
If .icon-link is intended to be a reusable component, creating new classes for these elements and adding styles to them, instead of nesting + requiring specific elements in the CSS, would make them a little more flexible and extensible (e.g. '.icon-link-anchor' instead of '.icon-link a').  Not sure if .icon-link is a new addition to this page or not; if it's not, it's probably not worth updating at this point.